src/SwedbankPay.Sdk/Models/Vipps/TransactionAPI/Response/AllTransactionResponseContainer.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace PayEx.Client.Models.Vipps.TransactionAPI.Response
+﻿namespace SwedbankPay.Sdk.Models.Vipps.TransactionAPI.Response
```

Comment:
Why the name `TransactionAPI`? I would like this namespace to read:

```suggestion
namespace SwedbankPay.Sdk.Vipps.Transactions
```

Or perhaps without `Transactions` altogether and just `SwedbankPay.Sdk.Vipps`.